src/web/app/pages-instructor/instructor-search-page/instructor-search-bar/instructor-search-bar.component.html

```diff
@@ -1,13 +1,13 @@
 <div class="card bg-light mb-4">
   <div class="card-body">
     <div class="text-muted">
-      You can search for students from all your courses here.
+      You can search for students (by name or email) from all your courses here.
```

Comment:
The search bar can actually search for more fields according to the Help page. i.e. Section name, Team name. I know the issue suggested name and email, but would it be better if we could indicate all the fields to be searched?

Reply:
Update: I added all fields (section name, team name, student name, and email) that can be used to search in the search bar and pushed those changes. Now, I am waiting to see if all checks pass.